lib/vanagon/driver.rb

```diff
@@ -16,6 +16,9 @@ class Driver
     def initialize(platform, project, options = { :configdir => nil, :target => nil, :engine => nil, :components => nil, :skipcheck => false })
       @verbose = false
       @preserve = false
+      @workdir = options[:workdir]
+      remote_workdir = options[:remote_workdir]
```

Comment:
Nitpick: I'd continue to assign to the instance variable rather than the accessor (for consistency). It doesn't have to be fixed up before this is merged, but we'll probably fix it up immediately after we merge it.